client/src/exampleData.js

```diff
@@ -0,0 +1,17 @@
+
+let exampleItem = {
+  itemDescription: ['here is an item', 'item description', 'random text', 'this is a test data'],
+  itemName: 'ECHO 4th gen white',
+  productId: 1089,
+  itemColor: 'white',
+  similarItems: [1087, 1088],
+  configuration: ['no echo', '1st gen', 'with bulbs on it mounted'],
+  brand: 'Amazon',
+  isFreeDelivery: true,
+  isPrimeFreeOneDay: true,
+  ratings: 20500,
+  price: 70,
+  answeredQuestions: 435
+};
+
+export default exampleItem;
```

Comment:
This is more of a question rather than a comment. Are you considering how you're going to render data received from other services and rendering them into your components? Like for instance I have the item's price but in your example data you include price. Is this purely just to make sure some price renders?